recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.03.18" %}
+{% set version = "2018.04.03" %}
```

Comment:
perhaps it is best to leave version bumps out until some tag? It seems like this will be inaccurate, because it really should be when the PR gets merged, not proposed, and it also seems like it will be highly prone to creating merge conflicts.

Reply:
Per our discussion earlier, I'll drop the date change.